docs/tutorials/infer_phys_pars.rst

```diff
@@ -0,0 +1,503 @@
+
+
+Given the effective distance, it is possible to derive a relation between
+the distance to the pulsar :math:`d_\mathrm{p}` and the distance to the screen
+:math:`d_\mathrm{s}`. In terms of the fractional screen--pulsar distance
+:math:`s`, the two true distances are given by
+
+.. math::
+
+    d_\mathrm{p} &= \frac{ s }{ 1 - s } d_\mathrm{eff}, \\
+    d_\mathrm{s} &= s d_\mathrm{eff}.
+
+.. jupyter-execute::
+
+    ns = 250
+    s = np.arange(0.5/ns, 1., 1./ns)
+
+    d_p = s / (1. - s) * d_eff
+    d_s = s * d_eff
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(7., 6.))
+
+    plt.plot(s, d_p.to(u.pc), label=r'pulsar distance $d_\mathrm{p}$')
+    plt.plot(s, d_s.to(u.pc), label=r'screen distance $d_\mathrm{s}$')
+
+    plt.yscale('log')
+
+    plt.xlim(0., 1.)
+    plt.ylim(1., 1.e5)
+
+    plt.legend(loc='upper left')
+
+    plt.xlabel(r'fractional screen-pulsar distance $s$')
+    plt.ylabel(r'distance from Earth (pc)')
+
+    plt.show()
+
+This also shows that the effective distance sets a maximum on the distance to
+the screen :math:`d_\mathrm{s} < d_\mathrm{eff}`.
+
+
+Pulsar distance--orbital inclination relation
+---------------------------------------------
+
+The aplitude of the pulsar's orbital velocity signature :math:`A_\mathrm{p}`
+can be used to derive a relation between the distance to the pulsar system
+:math:`d_\mathrm{p}` and the sine of its orbital inclination
+:math:`\sin( i_\mathrm{p} )`, following
+
+.. math::
+
+    d_\mathrm{p} &= \frac{ \sqrt{ d_\mathrm{eff} } }{ A_\mathrm{p} }
+                    \frac{ K_\mathrm{p} }{ \sin( i_\mathrm{p} ) }
+                    b_\mathrm{p} \\
+                 &= \frac{ v_\mathrm{orb,E} K_\mathrm{p} }
+                         { A_\mathrm{E} A_\mathrm{p} }
+                    \frac{ b_\mathrm{E} b_\mathrm{p} }{ \sin( i_\mathrm{p} ) }.
+
+.. jupyter-execute::
+
+    nsini_p = 400
+    sini_p = np.arange(0.5/nsini_p, 1., 1./nsini_p)
+
+    b2_p = (1 - sini_p**2) / (1 - sini_p**2 * np.cos(chi_p)**2)
+    d_p = v_orb_e * k_p / (amp_e * amp_p) * np.sqrt(b2_e * b2_p) / sini_p
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(7., 6.))
+
+    plt.plot(sini_p, d_p.to(u.pc))
+
+    plt.yscale('log')
+
+    plt.xlim(0., 1.)
+    plt.ylim(10., 1.e5)
+
+    plt.xlabel(r"sine of pulsar's orbital inclination $\sin( i_\mathrm{p} )$")
+    plt.ylabel(r"pulsar's distance from Earth $d_\mathrm{p}$ (pc)")
+
+    plt.show()
+
+
+The lens velocity
+-----------------
+
+Finally, it is possible to find a constraint on the projected lens velocity
+:math:`v_\mathrm{lens}`. This is best expressed in terms of some intermediate
+quantities derived above (:math:`\xi` and :math:`d_\mathrm{eff}`) and as a
+function the fractional screen--pulsar distance :math:`s`:
+
+.. math::
+
+    v_\mathrm{lens} = s \left( v_\mathrm{p,sys,eff}
+                               \pm \sqrt{ d_\mathrm{eff} } C \right),
+    \qquad \mathrm{with} \qquad
+    v_\mathrm{p,sys,eff} \simeq d_\mathrm{eff}
```

Comment:
Why `\simeq`?  Also, maybe useful to write here as `= d_{\rm eff} \mu_{\rm eff} = ` to make clear how the projection works.

Reply:
I use ``\simeq`` for "is asymtotically equal to". So here I use it because of the small angle approximation. If you think it causes more confusion than clarity, we can change it to just ``=`` (here and elsewhere in the tutorials).

Maybe the quantity ``v_p_sys_eff`` I introduced is confusing. It denotes the unsigned contribution of the pulsar's systemic motion to the effective velocity, i.e., ``((1-s)/s) * v_p_sys``.

Let's double-check if I got this right, but I think ``v_p_sys_eff`` does not equal ``d_eff * mu_eff``, but rather ``d_eff * mu_p_sys`` (since ``mu_eff = v_eff/d_eff`` per your derivations page).

I made some changes here and in the "model parameters" section. See if this made things more clear and understandable, or if I made it worse...